www/student.html

```diff
@@ -22,14 +22,9 @@ <h1>Input:</h1>
         <button id="end-button" disabled>End</button>
         <p id="time-displayed"></p>
         <h1>Code Output:</h1>
-        <textarea id="code-output"></textarea>
+        <textarea class="textArea" id="code-output"></textarea>
```

Comment:
Do we need a separate `textArea` class, or can styles be applied to all `<textarea>` elements on the page?